adb/adb_debug.py

```diff
@@ -27,12 +27,13 @@
 
 try:
   import sign_m2crypto
+  rsa_signer = sign_m2crypto.M2CryptoSigner
 except ImportError:
-  sign_m2crypto = None
-try:
-  import sign_pythonrsa
-except ImportError:
-  sign_pythonrsa = None
+  try:
+    import sign_pythonrsa
+    rsa_signer = sign_pythonrsa.PythonRSASigner.FromRSAKeyPath
```

Comment:
`rsa_signer = sign_pythonrsa.PythonRSASigner`
would work.
You don't actually need to modify sign_pythonrsa then.